### ISSUE TITLE
Unable to determine geometry

### ISSUE THREAD

**Issue by mfraser** [stardot/b-em#71]
A lot of the downloaded .ssd images I have are failing to load into b-em with the error Unable to determine geometry. The same file loads without issue in beebem.

**Comment by SteveFosdick** [comment]
Could you attach one such problematic image, please?

**Comment by mfraser** [comment]
[Arcadians.ssd.zip](https://github.com/stardot/b-em/files/2356550/Arcadians.ssd.zip)

**Comment by SteveFosdick** [comment]
I have re-worked the the way discs are recognised in this commit: https://github.com/stardot/b-em/commit/d11b167d188fd64e3f9ea3b50f1d3ddfd474adc7

Once I have had a change to do some more testing on it I'll push it to master.

**Comment by SteveFosdick** [comment]
This took a slightly different turn and the latest solution uses a combination of file extension and some auto-detection.  It detects the disc in the report correctly as well as one in a previous similar but and all the SSDs in the BEEB.MMB file referred to here: https://stardot.org.uk/forums/viewtopic.php?f=4&p=218216#p218216.

I have merged the commit concerned to master so this is now fixed in the master branch.
